packages/databricks-sdk-js/src/config/BricksCliCredentials.ts

```diff
@@ -0,0 +1,93 @@
+import {Provider} from "../types";
+import {
+    AuthType,
+    Config,
+    ConfigError,
+    CredentialProvider,
+    RequestVisitor,
+} from "./Config";
+import {execFile, isFileNotFound} from "./execUtils";
+import {refreshableTokenProvider, Token} from "./Token";
+
+/**
+ * Authenticate through the Bricks CLI.
+ */
+export class BricksCliCredentials implements CredentialProvider {
+    public name: AuthType = "bricks-cli";
+
+    async configure(config: Config): Promise<RequestVisitor | undefined> {
+        if (!config.isAws()) {
+            return;
+        }
+
+        if (config.host === "") {
+            return;
+        }
+
+        const ts = this.getTokenSource(config);
+        try {
+            await ts();
+        } catch (e) {
+            if (e instanceof ConfigError) {
+                if (e.message.indexOf("Can't find 'bricks' command") >= 0) {
+                    config.logger.debug(
+                        "Most likely Bricks CLI is not installed"
```

Comment:
please add a corner case when oauth is not configured or not supported

https://github.com/databricks/databricks-sdk-go/blob/main/config/auth_u2m.go#L32-L35

Reply:
Thanks, added.